### ISSUE TITLE
Dynamic LIKE is Impossible

### ISSUE THREAD

**Issue by timscott** [knex/knex#3564]
# Environment

Knex version: 0.20.2
Database + version: MS SQL Server

# Bug

It seems there is no way to do a dynamic like.

```javascript
const q = 'foo';
const term = `%${q}%`;
.whereRaw('target_name like ?', [term]) // rest of the query omited for clarity
```
Logs this:

> knex:bindings [ 'undefinedoolundefined] undefined

I tried many other ways:

```.whereRaw('target_name like ?', [`\%${q}\%`])```
> knex:bindings [ 'NaNoo%']

And:

```.where('target_name', 'like', `%${q}%`)```
>  knex:bindings [ 'NaNoo%' ]

And:

```.whereRaw(`target_name like '%${q}%'`)```
> where target_name like 'NaNoo%'

And:

````.whereRaw(`target_name like '\%${q}\%'`)````
> where target_name like 'NaNoo%'

If it's not a bug, I guess the documentation on how to do this is missing or hard to find.

**Comment by timscott** [comment]
Upon further investigation, the bug is in the logging. The [utility](https://github.com/visionmedia/debug/blob/825d35a2da0a9fa115baffea2f07d22b876332fc/src/common.js#L67) that formats output for logging makes a mess of percents.

**Comment by kibertoad** [comment]
Released in 0.20.4
